git-send-email.perl

```diff
@@ -601,14 +615,14 @@ sub is_format_patch_arg {
 	if ($f eq "--") {
 		push @rev_list_opts, "--", @ARGV;
 		@ARGV = ();
-	} elsif (-d $f and !is_format_patch_arg($f)) {
+	} elsif (-d $f and (!is_format_patch_arg($f))) {
```

Comment:
It is well-established in Git's source code that `!` binds stronger than `and`. I'd vote to leave this line as-is.